go/libraries/doltcore/env/actions/reset.go

```diff
@@ -136,26 +136,21 @@ func ResetHardTables(ctx context.Context, dbData env.DbData, cSpecStr string, ro
 	return resetHardTables(ctx, dbData, cSpecStr, roots)
 }
 
-func ResetHard(ctx context.Context, dEnv *env.DoltEnv, cSpecStr string, roots doltdb.Roots) error {
+func ResetHard(ctx context.Context, dEnv *env.DoltEnv, cSpecStr string, roots doltdb.Roots, headRef *ref.DoltRef, ws *doltdb.WorkingSet) error {
```

Comment:
(minor) Would be nice to add some godocs for this function, especially since the args list is getting longer and it's a public/exposed function from this package.

Reply:
Docs added.